roles/kubernetes/node/tasks/main.yml

```diff
@@ -113,10 +113,7 @@
     name: "{{ item }}"
     state: present
   with_items:
-    - ip_vs
-    - ip_vs_rr
-    - ip_vs_wrr
-    - ip_vs_sh
+    - "{{ kube_proxy_ipvs_modules }}"
```

Comment:
```suggestion
```

Reply:
Hi , this change i have not add to my commit as I don't think i understand it